### PR TITLE
PubSubHubbub

### PR DESCRIPTION
**Pull request by Alkarex**
https://github.com/FreshRSS/FreshRSS/issues/312
http://blog.superfeedr.com/howto-pubsubhubbub/

### CODE REVIEW

**Comment by Alkarex** [comment]
I have done a special effort to make only very few changes to the FreshRSS code when adding PubSubHubbub, so the pull request is relatively small compared to the impact of this new feature.

**Comment by julien51** [comment]
+1

File: app/i18n/cz/conf.php [hunk 1]
```diff
@@ -84,6 +84,7 @@
 		'articles_per_page' => 'Počet článků na stranu',
 		'auto_load_more' => 'Načítat další články dole na stránce',
 		'auto_remove_article' => 'Po přečtení články schovat',
+		'mark_updated_article_unread' => 'Označte aktualizované položky jako nepřečtené',
```
Comment (Alkarex) [inline app/i18n/cz/conf.php:87]: @Tets42 Please check

File: app/i18n/cz/sub.php [hunk 2]
```diff
@@ -37,6 +37,7 @@
 		'url' => 'URL kanálu',
 		'validator' => 'Zkontrolovat platnost kanálu',
 		'website' => 'URL webové stránky',
+		'pubsubhubbub' => 'Okamžité oznámení s PubSubHubbub',
```
Comment (Alkarex) [inline app/i18n/cz/sub.php:40]: @Tets42 Please check

File: app/i18n/de/sub.php [hunk 3]
```diff
@@ -37,6 +37,7 @@
 		'url' => 'Feed-URL',
 		'validator' => 'Überprüfen Sie die Gültigkeit des Feeds',
 		'website' => 'Webseiten-URL',
+		'pubsubhubbub' => 'Sofortige Benachrichtigung mit PubSubHubbub',
```
Comment (Alkarex) [inline app/i18n/de/sub.php]: @thomasE1993 Please check

Comment (ghost) [inline app/i18n/de/sub.php]: Je préfère 'Sofortbenachrichtigung mit PubSubHubbub' ou 'Sofortbenachrichtigung durch PubSubHubbub'

Comment (Alkarex) [inline app/i18n/de/sub.php]: Merci, c'est fait https://github.com/Alkarex/FreshRSS/commit/d3af903301ce2b59d2720e94727945cf9ff31a40 :-)

**Comment by Alkarex** [comment]
I suggest leaving this pull request open till after we release beta 1.1.1, so that we can use the current /dev to release beta 1.1.1. We can target beta 1.1.2 next month for PubSubHubbub.

**Comment by Alkarex** [comment]
There is apparently a problem with the feed http://rss.slashdot.org/Slashdot/slashdot, which uses the hub http://pubsubhubbub.appspot.com/ . I receive a "204" status code during the subscription, which is fine, but then never receive any post.

**Comment by Alkarex** [comment]
Merging the PubSubHubbub code to start testing in /dev towards next FreshRSS version 1.1.2

**Pull request merged**